### PR TITLE
Feature/123 fix manage bookings

### PR DESCRIPTION
**Pull request by TobiasABay**

### CODE REVIEW

**Review by Androkless — APPROVED** [review]
Commented stuff killed

**Review by Androkless — APPROVED** [review]
Commented stuff killed

**Review by Androkless — APPROVED** [review]

File: src/pages/RoomOverviewPage.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import FullCalendar from '@fullcalendar/react';
 import resourceTimelinePlugin from '@fullcalendar/resource-timeline';
-import { LinearProgress } from '@mui/material';
+// import { LinearProgress } from '@mui/material';
```
Comment (MathiasFM) [inline src/pages/RoomOverviewPage.tsx]:
```suggestion

```
Slet dette i stedet for at kommentere det ud 🦧

File: src/pages/RoomOverviewPage.tsx [hunk 2]
```diff
@@ -66,21 +66,19 @@ const MyFullCalendarComponent = () => {
 		setEvents(newEvents);
 	}, [roomsWithBookings]);
 
-	if (allBookings.isLoading) {
-		return <LinearProgress />;
-	}
+	// if (allBookings.isLoading) {
+	// 	return <LinearProgress />;
+	// }
```
Comment (MathiasFM) [inline src/pages/RoomOverviewPage.tsx]:
```suggestion

```
Please slet i stedet for at kommenter ud 👍

**Review by MathiasFM — CHANGES_REQUESTED** [review]

**Review by MathiasFM — APPROVED** [review]
LGTM

**Pull request merged**